src/Surfnet/StepupRa/RaBundle/Controller/RaManagementController.php

```diff
@@ -91,9 +92,12 @@ public function raCandidateSearchAction(Request $request)
 
         $logger->notice(sprintf('Searching for RaCandidates within institution "%s"', $institution));
 
+        $pageNumber = $request->get('p', 1);
+        Assertion::digit($pageNumber, 'Expected page number to be an integer, got "%s"');
+
         $command                 = new SearchRaCandidatesCommand();
         $command->institution    = $institution;
-        $command->pageNumber     = $request->get('p', 1);
+        $command->pageNumber     = $pageNumber;
```

Comment:
Yes. It was supposed to be converted to an integer. I must have messed something up when git patching and ordering my additions.

The assertion is there to check whether a valid digit has been given in the string representing a page number before we convert it to an integer which the command requires. We merely check whether we can reliably parse the given string as an integer. I am opposed to changing the page number in the command to a string because I think a number should be modelled as an integer. Practically, it would also mean changing an external library, which could affect other dependents. 

I have to admit, I did not check for whether the status code / error page remains the same, perhaps it would be better to throw an application level exception which is guaranteed to be translated to the relevant HTTP code by Symfony.